packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -211,7 +213,6 @@ export const RowsList = (props: RowsListProps) => {
       if (rowHighlightIndex !== undefined && row.index === rowHighlightIndex) {
         style = { ...style, backgroundColor: theme.components.table.rowHoverBackground };
```

Comment:
```suggestion
        style = { ...style, backgroundColor: theme.colors.background.secondary };
```
Should we change the color so there's some differentiation between "hover" and "scrolledTo"?
<img width="1443" alt="image" src="https://github.com/grafana/grafana/assets/109082771/aa5b1476-6273-442f-b871-635b06bd35cc">

Although maybe that isn't enough, can't tell which is which in light mode: 
<img width="1405" alt="image" src="https://github.com/grafana/grafana/assets/109082771/cc91904e-3306-4567-a589-77b3694efb13">

Reply:
Deal with this in another issue, don't increase the scope of this pr